apps/frontend/src/components/containers/Auth/Shop/ItemsShowcase/ItemsShowcase.view.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import type { TItem } from '@/types/item';
+import classes from './ItemsShowcase.module.scss';
+
+type TProps = {
+	readonly allItems: TItem[];
+};
+
+const ItemsShowcaseView = (props: TProps) => {
+	return (
+		<div className={classes['container']}>
+			{props.allItems.map((item, index) => (
```

Comment:
if you don't have one currently, add `id` for each item from the BE
```suggestion
			{props.allItems.map((item) => (
			<div className={classes['itemContainer']} key={item.id}>
```